README.md

```diff
@@ -1 +1,81 @@
-# th2-conn-ws-client
\ No newline at end of file
+# WebSocket Client
+
+This microservice allows sending and receiving of messages via WebSocket protocol
+
+## Configuration
+
+Main configuration is done via setting following properties in a custom configuration:
+
++ **uri** - connection URI
++ **sessionAlias** - session alias for incoming/outgoing TH2 messages (e.g. `ws_api`)
++ **handlerSettings** - WebSocket event handler settings
+
+### Event handler configuration
+
+Event handler can be configured via setting following properties in the `handlerSettings` block of the main configuration
```

Comment:
Event handler can be configured via **settings** following **the** properties **inside** the `handlerSettings` block of the main configuration

Reply:
Same as above

Reply:
Please disconsider this comment as well